pkg/inspector/image-inspector.go

```diff
@@ -99,46 +100,58 @@ func NewDefaultImageInspector(opts iicmd.ImageInspectorOptions) ImageInspector {
 
 // Inspect inspects and serves the image based on the ImageInspectorOptions.
 func (i *defaultImageInspector) Inspect() error {
-	client, err := docker.NewClient(i.opts.URI)
-	if err != nil {
-		return fmt.Errorf("Unable to connect to docker daemon: %v\n", err)
-	}
+	var scanReport []byte
+	var htmlScanReport []byte
+	var scanner iiapi.Scanner
 
-	if err = i.pullImage(client); err != nil {
-		return err
-	}
+	switch i.opts.ScanType {
+	case "openscap":
+		client, err := docker.NewClient(i.opts.URI)
```

Comment:
Yeah, I did that just for the WIP to save time in the refactor since I was using a dummy file in the scanner.  The goal would be that everything just cares about the file system once it gets to the scanner and how that file system is produced is out of scope.